mimic/test/test_loadbalancer.py

```diff
@@ -1024,6 +1024,52 @@ def test_updating_node_success(self):
 
         self.assertEqual(self._get_nodes(self.lb_id)[0], expected)
 
+        # check if feed is updated
+        d = request(
+            self, self.root, "GET",
+            "{}/loadbalancers/{}/nodes/{}.atom".format(self.uri, self.lb_id,
+                                                       self.node[0]["id"]))
+        feed_response = self.successResultOf(d)
+        self.assertEqual(feed_response.code, 200)
+        self.assertEqual(
+            self.successResultOf(treq.content(feed_response)),
+            ("<feed xmlns=\"http://www.w3.org/2005/Atom\"><entry>"
+             "<summary>Node successfully updated with address: '127.0.0.1', "
+             "port: '80', weight: '100', condition: 'DISABLED'</summary>"
+             "<updated>1970-01-01T00:00:00.000000Z</updated></entry></feed>"))
```

Comment:
I think its fine to have this here since checking that feed is updated is an extension of this particular test. If if I add another positive test, I'll have to perform the steps done in this test (i.e. update node).